src/components/tabs/tab/tab.style.ts

```diff
@@ -17,7 +17,9 @@ const StyledTab = styled.div<StyledTabProps>`
 
       ${position === "left" &&
       css`
-        width: 80%;
+        margin-left: -6px;
+        margin-top: 6px;
+        width: calc(80% + 6px);
```

Comment:
**suggestion(non-blocking):** in `tab-title.style` you used design token values when applying styles. 6px is equivalent to `spacing075`. Would that work here too?

Reply:
Yep, will update to that, will the token just work correctly in a maths value? Ill test it out

Reply:
I'd imagine it would still work. A variable would work so I (rightly or wrongly) assume `styled-components` would treat it the token the same way as a variable.